UBCMint/moss
# Monaabusharkh18/neu 31 delete one user

Here give examples of the changes you've made in this pull request. Include an itemized list if you can.

• Created api route that delete a user based on its id, given the user exists and id is valid.
•  Returns a 200 or 405 or 400 or 500 error code signaling success or failure.

How to Test?

•Create DELETE request in Postman with the API endpoint http://localhost:3000/server/api/user/deleteuser/[id]

Should give this output if successful
<img width="897" alt="Screenshot 2024-03-19 at 2 36 58 PM" src="https://github.com/UBCMint/MINT-Open-Source/assets/84531176/cdf09bce-e95a-42fd-95d8-bd28f64a2cb3">

Invalid Id
<img width="1057" alt="Screenshot 2024-03-19 at 2 51 49 PM" src="https://github.com/UBCMint/MINT-Open-Source/assets/84531176/e276c56f-55d6-495c-b409-96010794d4ad">
